applications/nrf_desktop/src/hw_interface/Kconfig.motion

```diff
@@ -123,6 +123,14 @@ config DESKTOP_MOTION_SENSOR_SLEEP3_TIMEOUT_MS
 	  Time in milliseconds after last motion detection in which sensor
 	  enters a low power mode 3.
 
+config DESKTOP_MOTION_SENSOR_SLEEP_DISABLE_ON_USB
+	bool "Disable low power modes if USB connected"
+	depends on DESKTOP_MOTION_SENSOR_ENABLE
```

Comment:
depends on USB and battery charging (in case we will have USB enabled mouse but normal battery)

Reply:
Ok